doc/install.rst

```diff
@@ -76,7 +76,7 @@ First, we must configure conda to get packages from the
 Now we can create a new conda environment with Python and all our dependencies installed
 (we'll call it ``pygmt`` but you can change it to whatever you want)::
 
-     conda create --name pygmt python=3.8 pip numpy pandas xarray netcdf4 packaging gmt
+     conda create --name pygmt python=3.9 pip numpy pandas xarray netcdf4 packaging gmt
```

Comment:
https://docs.anaconda.com/anaconda/packages/pkg-docs/ 

It seems that the default Python version from Anaconda is still Python 3.8. Perhaps we should still use Python 3.8 here?

Reply:
Yep, sounds good.

```suggestion
     conda create --name pygmt python=3.8 pip numpy pandas xarray netcdf4 packaging gmt
```